mldsa/native/aarch64/meta.h

```diff
@@ -77,7 +77,7 @@ static MLD_INLINE int mld_rej_uniform_eta2_native(int32_t *r, unsigned len,
   MLD_CT_TESTING_DECLASSIFY(buf, buflen);
   outlen =
       (int)mld_rej_uniform_eta2_asm(r, buf, buflen, mld_rej_uniform_eta_table);
-  MLD_CT_TESTING_SECRET(r, sizeof(int32_t) * outlen);
+  MLD_CT_TESTING_SECRET(r, sizeof(int32_t) * (unsigned)outlen);
```

Comment:
Can this be addressed by declaring `outlen` as `unsigned` instead and casting it to `int` in the end? When that happens, a comment should be added why it is safe (we should have such for L79 already, where the conversion currently happens).